src/routes/index.ts

```diff
@@ -386,6 +387,8 @@ router.route(config.UPDATE_BENEFICIAL_OWNER_STATEMENTS_URL)
 
 router.get(config.OVERSEAS_ENTITY_PAYMENT_WITH_TRANSACTION_URL, authentication, companyAuthentication, overseasEntityPayment.get);
 
+router.get(config.UPDATE_PAYMENT_FAILED_URL, authentication, updatePaymentFailed.get);
```

Comment:
need companyAuthentication after authentication,

Reply:
which would need testing thoroughly. I'm assuming the user is authenticated for the company when the re-direct happens. But may be more important to show error page in all cases. Not sure now.

Reply:
Better to show failure.